.github/workflows/prestocpp-linux-adapters-build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
   prestocpp-linux-adapters-build:
     runs-on: ubuntu-22.04
     container:
-      image: prestodb/presto-native-dependency:0.292-20250204112033-cf8ba84
+      image: ghcr.io/czentgr/prestissimo-deps:centos9
```

Comment:
I believe this is a temporary solution--can you add a comment here describing why we need to use @czentgr's personal slug?

Reply:
I added a comment in the PR description. Once the PR lands, we will replace it with the official CI image from the release team. Its sort of a chicken and egg problem.
I will add a comment here.